schemas/bolt-yaml-plan.schema.json

```diff
@@ -0,0 +1,293 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Bolt YAML Plan",
+  "description": "Bolt YAML Plan Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "description": {
```

Comment:
Does this make `description` required for plans?

Reply:
No, only properties listed under `:required` are required. The `:properties` key just defines _known_ properties for a hash value.

Reply:
Realizing we probably want `required: ["steps"]` at that level, though.